package.json

```diff
@@ -21,13 +21,15 @@
     "@angular/platform-browser-dynamic": "~9.0.2",
     "@angular/router": "~9.0.2",
     "@angular/service-worker": "~9.0.2",
+    "angulartics2": "^8.3.0",
     "nhsuk-frontend": "^3.0.3",
     "rxjs": "~6.5.4",
     "tslib": "^1.10.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.900.3",
+    "@angular-devkit/build-ng-packagr": "~0.900.3",
```

Comment:
as discussed it will be good if you can update the ng start and build scripts to reflect the shared modules consumption

Reply:
excellent. will do